server/game/cards/06-TLS/AsakichiCooke.js

```diff
@@ -0,0 +1,40 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class AsakichiCooke extends DudeCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Move Your Dude',
+            cost: ability.costs.discardFromHand(),
+            target: {
+                activePromptTitle: 'Choose one of your other dudes at this location to move',
```

Comment:
Let's use less descriptive titles, you should know what you are doing when triggering a card.
e.g. "Choose your dude at this location"